pypicloud/storage/gcs.py

```diff
@@ -78,6 +80,9 @@ def _subclass_specific_config(cls, settings, common_config):
             "service_account_json_filename": service_account_json_filename,
             "project_id": settings.get("storage.gcp_project_id"),
             "use_iam_signer": asbool(settings.get("storage.gcp_use_iam_signer", False)),
+            "service_account_email": settings.get(
+                "storage.service_account_email", False
```

Comment:
can we name it "iam_signer_service_account_email" ?
People may get confused that this is the same service account which is used to access GCS. 

Ideally there are two service accounts
1. Svc account to access gcs
2. Svc account to sign gcs links

(As a fallback I was suggesting to reuse service account # 1 , when # 2 is not explicitly provided.)

Logic should be
1. Use json file for gcs service acount
2. use service account email "iam_signer_service_account_email" for signing.
   if not provided then fallback to the email of json file